openshift/templates/httpd.json

```diff
@@ -45,7 +45,10 @@
       "kind": "Route",
       "apiVersion": "v1",
       "metadata": {
-        "name": "${NAME}"
+        "name": "${NAME}",
+        "annotations": {
+          "template.openshift.io/expose-uri": "http://{.spec.host}{.spec.path}"
```

Comment:
@jerboaa: please see https://trello.com/c/PFfBJOsO and https://github.com/openshift/origin/pull/14486 ; am currently writing the docs for this...